specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/examples/capacityReservationExamples/CapacityReservationGroups_GetCapacityReservationGroupResourceIds.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can you run prettier and fix this?

Reply:
ran prettier and fixed the issues. please take a look. thanks